shelley/design-spec/delegation_design_spec.tex

```diff
@@ -1363,21 +1293,21 @@ \subsubsection{Operational Key Certificates}
 but not including these next 10 block headers. So it cannot rely on
 any information in the 10 corresponding block bodies.
 
-Without this information, the validity of the staking keys that signed
+Without this information, the validity of the hot key that signed
 the headers cannot be verified. If the network layer sees one of the
-10 new blocks signed by a key it doesn't recognise, it might be
+10 new blocks signed by a hot key it doesn't recognise, it might be
 because there is a delegation certificate in the block bodies (which
 the network layer has not seen yet), that shows that the key is valid
-because some other key deferred its staking rights to it. Similarly,
-if the network layer sees a known staking key, how can it know that it
+because some stake pool key deferred its staking rights to it. Similarly,
+if the network layer sees a known hot key, how can it know that it
 is still valid? There could be a new certificate in the block bodies
 that defers the rights of this key to a different one (which would
 invalidate the key the network layer saw).
 
 To address this problem, a new type of certificate is introduced:
 \emph{operational key certificates}. These certificates are provided
 in the block header itself, as part of the witness, thus solving the
-problem of determining the validity of staking keys. In other words,
+problem of determining the validity of hot keys. In other words,
```

Comment:
Here, we had stake addresses, now we have hot keys.  Is that a fundamental change or just terminology (i.e. are these hot stake keys, hot pool keys, or what?).   Also, I would think that a valid hot key is one that has been created from a cold key and is within its period of validity.  Is that what is meant, or something else?

Reply:
Re. 1: basically, we've introduced more types to the delegation system as we progressed with the design. At first, we had stake keys for different things, and delegations between them, and special rules restricting chain delegations. Now, we have stake addresses, which delegate to pool keys, which issue certificates for hot keys (and no special rules restricting chain delegations, since there is only one possible chain respecting the types).

Re. 2: what I really meant is _precedence_ of hot keys, so I'll change that :)